DOCKER-network/dockernet

```diff
@@ -205,13 +227,13 @@ def update(arguments):
 def stop(arguments):
     print 'Stopping Docker Network...'
     # stop all nodes
-    os.system('docker kill $(docker ps -q -f network=docker-network_vpcbr)')
```

Comment:
just a simple question, is this `docker-network_vpcbr` the name for the network?
because on my local it's `protocol_vpcbr`:
```
❯ docker network ls                                                                                ─╯
NETWORK ID          NAME                DRIVER              SCOPE
e79d007d3dc4        bridge              bridge              local
acd9abe4dd4d        host                host                local
9e541f3928d3        none                null                local
2e7661fe9460        protocol_default    bridge              local
4f8447bf3df3        protocol_vpcbr      bridge              local
```
maybe mine branch is outdated, just want to confirm